src/main/resources/sprache/English.properties

```diff
@@ -1147,14 +1147,17 @@ TotalEPV=Total team value (EPV)
 dbcleanup.cleanupnow=Cleanup Now
 dbcleanup.none=Remove none
 dbcleanup.all=Remove all
+dbcleanup.allTime=Remove all time
 dbcleanup.removeOlderThan=Remove older than
 dbcleanup.weeks=weeks
-dbcleanup.ownMatches=Own matches
-dbcleanup.ownFriendlies=Own friendlies
-dbcleanup.otherMatches=Other matches
-dbcleanup.otherFriendlies=Other friendlies
+dbcleanup.yourMatches=Your matches
+dbcleanup.yourFriendlies=Your friendlies
+dbcleanup.otherTeamsMatches=Other teams' matches
+dbcleanup.otherTeamsFriendlies=Other teams' friendlies
 dbcleanup.hrf=Remove HRFs
-dbcleanup.hrfAutoremove=Remove HRFs intelligent (keep only one HRF per week)
+dbcleanup.hrfSmartRemove=Smart Delete of HRFs (keep only one HRF per week)
```

Comment:
Changing the keys will invalidate the corresponding translations. Is that really desired at this point. Wouldn't it be sufficiant to change the values?

Reply:
I did wonder about this.  The reason I went for new keys was that the value for these entries has changed, so new translations are required in any case, whether we change the keys, or keep the same ones.  Is poeditor smart enough to request translators to provide  new translations for the keys that have changed?